modules/reformat_rhea_chebi/main.nf

```diff
@@ -0,0 +1,18 @@
+process REFORMAT_RHEA_CHEBI {
+    label 'light'
+    container 'quay.io/biocontainers/python:3.10'
+
+    input:
+    val rhea_rheactions_gz
+
+    output:
+    path 'rhea_chebi_mapping_*.tsv', emit: tsv_rhea_chebi_mapping
+
+    script:
+    
+    """
+    wget ${rhea_rheactions_gz}
```

Comment:
if that one is set in params, I guess you can just leave that module without input and use `params. rhea_chebi_download_mapping` directly in wget

Reply:
Done!